includes/Model/Product.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Class Product
+ *
+ * @link      https://github.com/googleforcreators/web-stories-wp
+ *
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ */
+
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Model;
+
+/**
+ * Class Product
+ */
+class Product {
+	/**
+	 * Product id.
+	 *
+	 * @var string
+	 */
+	protected $id;
+	/**
+	 * Product title.
+	 *
+	 * @var string
+	 */
+	protected $title;
+	/**
+	 * Product brand.
+	 *
+	 * @var string
+	 */
+	protected $brand;
+	/**
+	 * Product Price.
+	 *
+	 * @var string
+	 */
+	protected $price;
+	/**
+	 * Product's price currency.
+	 *
+	 * @var string
+	 */
+	protected $price_currency;
+	/**
```

Comment:
Do we maybe want to do something with the enum type here?

https://github.com/GoogleForCreators/web-stories-wp/pull/11250/files#diff-bad54f2ef7826324057d1449bd37884eab04ec70e335f6a9354850427a1d3102R28

I think it's fine as a string but might help for tooling etc..

Reply:
Any recommendation on to do this?

Reply:
Might be a bit overkill since we don't really modify the values but just pass them through.

Though theoretically we could take the currency list from the schema, store it in a const somewhere and then use `value-of<our_currency_const>`, which [PHPStan supports](https://phpstan.org/writing-php-code/phpdoc-types#key-and-value-types-of-arrays-and-iterables).

But again, probably a bit overkill.